frontend/src/components/Theaters/Settings.component.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <v-flex
+    xs-12
+    sm5
+  >
+    <v-chip
+      color="white"
+      text-color="black">
+      <v-icon left>build</v-icon>
+      Theater settings
+    </v-chip>
+    <v-chip
+      color="white"
+      text-color="black">
+      <span v-if="theater.kind==='m'">Type: cinema</span>
```

Comment:
nisam siguran da li imamo jos, ali trebalo bi napraviti model za theater, i onda na njemu imati funkciju koja to proverava, pa bi ovde bilo nesto tipa `theater.isM` (zaboravio sam sta je M :joy:)